user-interface/src/case-detail/panels/CaseDetailNavigation.test.tsx

```diff
@@ -44,29 +43,4 @@ describe('Navigation tests', () => {
 
     expect(result).toEqual(NavState.COURT_DOCKET);
   });
-
-  test('should render with css class name of "case-details-navigation fixed" when ref.fix() is called and "case-details-navigation" when ref.loosen() is called', async () => {
```

Comment:
I'm checking now to make sure we put a similar test in the CaseDetailScreen tests.

Reply:
The test `should fix when scrolled at top of viewport` on line 708 of CaseDetailScreen.test.tsx may be sufficient, but it doesn't have the same approach. @fmaddenflx, do you think we should have a test like this that calls the `fix()` and `loosen()` functions?